app/assets/mutations/createAsset.ts

```diff
@@ -69,16 +69,19 @@ export default resolver.pipe(
 
     // Notify recipients on Knock. This should be done asynchronously
     // (for example using background jobs, or other similar pattern)
-    await knockClient.notify("new-asset", {
-      actor: `${userId}`,
-      recipients,
-      data: {
-        asset_url: asset.url,
-        project_name: asset.project.name,
-        projectId: asset.project.id,
-      },
-    })
-
-    return asset
+    try {
```

Comment:
i wrapped all these in a try/catch because if you skip the README, don't add your workflows, and just start poking around, your comments/assets etc won't update if this call fails and it makes it feel like things are broken.